default/python/universe_generation/empires.py

```diff
@@ -98,10 +128,11 @@ def compile_home_system_list(num_home_systems, systems):
     min_jumps = min(10, max(int(float(len(systems)) / float(num_home_systems * 2)), 5))
     # try to find the home systems, decrease the min jumps until enough systems can be found, or the min jump distance
     # gets reduced to 0 (meaning we don't have enough systems to choose from at all)
+    additional_reqs = homesystem_vicinity_filter if _FILTER_HOMESYSTEMS_BY_VICINITY else None
```

Comment:
As explained by the variable name and the comment that goes with it, this is a parameter that controls the behavior-- when set to True the min planets req acts as a filter, when set to False the req acts as a post-selection modification.  If I simply used True and False, then there would be two different locations that had to be changed in synchrony in order to change the behavior.